src/identity/components/OrganizationListTab/index.tsx

```diff
@@ -0,0 +1,244 @@
+  }
+
+  const handleFilterOrgs = (searchTerm: string) => {
+    const url = new URL(location.href)
+    url.searchParams.set('searchTerm', searchTerm)
+    history.replaceState(null, '', url.toString())
+    setSearchTerm(searchTerm)
+    handleChangePage(1)
+  }
+
+  const handleSortOrgs = (
+    newSortKey: BucketSortKey,
+    newSortDirection: Sort
+  ) => {
+    const url = new URL(location.href)
+    url.searchParams.set('sortKey', newSortKey)
+    url.searchParams.set('sortDirection', newSortDirection)
+    history.replaceState(null, '', url.toString())
+
+    setSortMethod({
+      sortKey: newSortKey,
+      sortDirection: newSortDirection,
+      sortType: SortTypes.String,
+    })
+    handleChangePage(1)
+  }
+
+  // Load the orgs in the current account, if not yet loaded.
+  useEffect(() => {
+    if (orgsLoadedStatus === RemoteDataState.NotStarted) {
+      dispatch(getQuartzOrganizationsThunk(currentAccountId))
+    }
+  }, [currentAccountId, dispatch, orgsLoadedStatus])
+
+  // Determine whether to display the 'Upgrade' banner.
+  useEffect(() => {
+    const checkOrgCreationAllowance = async () => {
+      try {
+        const {allowed, availableUpgrade} = await fetchOrgCreationAllowance()
+
+        const newOrgAllowance = {
+          availableUpgrade: availableUpgrade,
+          isAtOrgLimit: !allowed,
+          status: RemoteDataState.Done,
+        }
+
+        setOrgAllowance(newOrgAllowance)
+      } catch (err) {
+        setOrgAllowance({...defaultOrgAllowance, status: RemoteDataState.Error})
+
+        dispatch(notify(fetchOrgAllowanceFailed()))
+      }
+    }
+
+    if (orgAllowance.status === RemoteDataState.NotStarted) {
+      checkOrgCreationAllowance()
+    }
+  }, [dispatch, orgAllowance.status])
+
+  // Load search, sort, and pagination settings from URL search params.
+  useEffect(() => {
+    const params = new URLSearchParams(window.location.search)
+    const urlSearchTerm = params.get('searchTerm')
+    const urlSortDirection = params.get('sortDirection') as Sort
+    const urlSortKey = params.get('sortKey')
+
+    const validSortDirections = [Sort.Ascending, Sort.Descending, Sort.None]
+
+    if (urlSearchTerm) {
+      setSearchTerm(urlSearchTerm)
+    }
+
+    if (
+      urlSortDirection &&
+      urlSortKey &&
+      validSortDirections.includes(urlSortDirection)
+    ) {
+      setSortMethod({
+        sortKey: urlSortKey,
+        sortDirection: urlSortDirection,
+        sortType: SortTypes.String,
+      })
+    }
+  }, [])
+
+  useEffect(() => {
+    const params = new URLSearchParams(window.location.search)
+    const urlPageNumber = parseInt(params.get('page'), 10)
+
+    if (typeof urlPageNumber === 'number' && urlPageNumber > 0) {
+      setCurrentPage(Math.min(urlPageNumber, totalPages))
+    }
+  }, [totalPages])
+
+  return (
+    <>
+      <FlexBox direction={FlexDirection.Row}>
+        <FlexBox className="account-settings-page-org-tab--searchwidget-container">
```

Comment:
Do you mean to have this `FlexBox` or did you mean to add a `FlexBox.Child`?

Reply:
💯 This should be a `FlexBox.Child`